src/registrar/forms/portfolio.py

```diff
@@ -160,3 +162,100 @@ class Meta:
             "roles",
             "additional_permissions",
         ]
+
+
+class NewMemberForm(forms.ModelForm):
+    member_access_level = forms.ChoiceField(
+        label="Select permission",
+        choices=[("admin", "Admin Access"), ("basic", "Basic Access")],
+        widget=forms.RadioSelect(attrs={"class": "usa-radio__input  usa-radio__input--tile"}),
+        required=True,
+        error_messages={
+            "required": "Member access level is required",
+        },
+    )
+    admin_org_domain_request_permissions = forms.ChoiceField(
+        label="Select permission",
+        choices=[("view_only", "View all requests"), ("view_and_create", "View all requests plus create requests")],
+        widget=forms.RadioSelect,
+        required=True,
+        error_messages={
+            "required": "Domain request permission is required",
+        },
+    )
+    admin_org_members_permissions = forms.ChoiceField(
+        label="Select permission",
+        choices=[("view_only", "View all members"), ("view_and_create", "View all members plus manage members")],
+        widget=forms.RadioSelect,
+        required=True,
+        error_messages={
+            "required": "Member permission is required",
+        },
+    )
+    basic_org_domain_request_permissions = forms.ChoiceField(
+        label="Select permission",
+        choices=[
+            ("view_only", "View all requests"),
+            ("view_and_create", "View all requests plus create requests"),
+            ("no_access", "No access"),
+        ],
+        widget=forms.RadioSelect,
+        required=True,
+        error_messages={
+            "required": "Member permission is required",
+        },
+    )
+
+    email = forms.EmailField(
+        label="Enter the email of the member you'd like to invite",
+        max_length=None,
+        error_messages={
+            "invalid": ("Enter an email address in the required format, like name@example.com."),
+            "required": ("Enter an email address in the required format, like name@example.com."),
+        },
+        validators=[
+            MaxLengthValidator(
+                320,
+                message="Response must be less than 320 characters.",
+            )
+        ],
+        required=True,
+    )
+
+    class Meta:
+        model = User
+        fields = ["email"]
+
+    def clean(self):
```

Comment:
Both approaches have merit.  One manually builds the error messages, the other subtracts from the automatically populated error list.  What I like about the subtractive method (at least once I add the appropriate checks) is in this case it seems simpler to maintain (easier to extend), since we let Django handle all the error handling except for those ones we want to hide.  I think I will keep this approach for this page.